doc/src/mysql.md

```diff
@@ -93,3 +87,15 @@ Enabling a Percona role first and only setting an initial script later won't hav
 any effect anymore.
 % hidden note as of 20240711: It is possible to re-trigger db initialisation by `touch /run/mysql_init`, but we have decided not to expose this as an official stable API.
 :::
+
+## Migrate user password hash algorithm
+
+Before Percona (and generally MySQL) 8.4, `mysql_native_password` was the default authentication and password hash algorithm.
```

Comment:
- mention that we switch off `mysql_native_password` in 25.11
- add sensu check that show up with a warning iff there are still users with `mysql_native_password`
  - consider backporting this check to 24.11

Reply:
We discussed that the check should be another PR.